circuits/package.json

```diff
@@ -28,9 +28,9 @@
   },
   "dependencies": {
     "circomlib": "https://github.com/weijiekoh/circomlib#ac85e82c1914d47789e2032fb11ceb2cfdd38a2b",
-    "maci-core": "^1.1.2",
-    "maci-crypto": "^1.1.2",
-    "maci-domainobjs": "^1.1.2",
+    "maci-core": "workspace:^",
```

Comment:
Have you tried publishing like this? I did once when using yarn workspaces, it published packages with this syntax and once you tried to install it from npm it failed saying it's not valid syntax

Reply:
Yes, we published packages using pnpm workspaces.
https://github.com/CryptKeeperZK/crypt-keeper-extension/blob/0e4e3a0ef85f533cc35c6174d8a6c25492d1ec33/packages/providers/package.json#L39

I can try to publish it locally to check if it works.